$OpenBSD: patch-tests_regression_tests_Makefile_am,v 1.1 2020/01/09 23:31:25 sthen Exp $

A couple of tests have missing files in the source distribution.

Index: tests/regression_tests/Makefile.am
--- tests/regression_tests/Makefile.am.orig
+++ tests/regression_tests/Makefile.am
@@ -1 +1 @@
-SUBDIRS = shapes vuln
+#SUBDIRS = shapes vuln
